packages/device_info/lib/device_info_tizen.dart

```diff
@@ -0,0 +1,92 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
If this file is for tizen, does not it need to add our copyright?

Reply:
done.